flutter/news/lib/features/frontpage/data/models/article_model.dart

```diff
@@ -4,13 +4,13 @@ import 'package:news/features/frontpage/domain/entities/article.dart';
 class ArticleModel extends Article {
   ArticleModel(
       {required SourceModel source,
-      required String author,
-      required String title,
-      required String description,
-      required String url,
-      required String urlToImage,
-      required String publishedAt,
-      required String content})
+      required String? author,
```

Comment:
are all this fields really optional?

Reply:
did some more testing on the api and removed some unnecessary` ?`

updated on [34d2f99](https://github.com/novoda/multiplatform-playground/pull/15/commits/34d2f99e34fbfc03733e882fdcc758565e92553f)